### PR TITLE
Added storage monitoring Jenkins job 

### PR DESCRIPTION
**Pull request by VermaSh**
This addresses [Storage Monitoring Jenkins job](https://github.com/AdoptOpenJDK/openjdk-jenkins-helper/issues/12)

- Implemented storage monitoring job for linux, aix, and mac
- Updated the readme to reflect the new Jenkins job

### CODE REVIEW

File: Jenkins_jobs/WorkspaceInfo.groovy [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    for (String line : subdirectoriesArray) {
+        if (!line.contains("tmp")) {
+            line = line.replaceAll("\\s+", "    ")
+            output += "\n    ${line}";
+        }
+    }
+    output += "\n=======================================================================\n\n";
+}
+
+parallel clones
```
Comment (AdamBrousseau) [inline Jenkins_jobs/WorkspaceInfo.groovy]: Please add a newline at the end of the file

File: Jenkins_jobs/WorkspaceInfo.groovy [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+/* Iterates over all the online nodes in Jenkins and
+ * prints contents of workspace folder along with
+ * the space they occupy. With exception of tmp directory
+ */
+node {
+    stage('Print_Space_Monitoring_Data') {
+        NodeHelper nodeHelper = new NodeHelper();
+
+        String projectLabel = params.projectLabel;
```
Comment (AdamBrousseau) [inline Jenkins_jobs/WorkspaceInfo.groovy]: If I don't define this parameter, or if I don't set it, will I iterate over all the nodes?

Comment (VermaSh) [reply Jenkins_jobs/WorkspaceInfo.groovy]: It won't iterate over any nodes. That's on my todo for after I have checked in rest of the jobs

File: Jenkins_jobs/WorkspaceInfo.groovy (hunk 1 above)
Comment (VermaSh) [reply Jenkins_jobs/WorkspaceInfo.groovy]: Done!

**Comment by gdams** [comment]
@AdamBrousseau are you happy with the PR now?

**Pull request merged**